_raml/apps/products/examples/patch_product_productid_customattributes_request.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+      "op": "add",
+      "path": "/",
+      "value":
+      {
+        "key": "Colour",
```

Comment:
Don't we write the API Docs in en-US?

Reply:
@AliceStatler We do. Unfortunately it's not consistent throughout the NOW API docs. But you're right, we should start heading that way. I'll change that.